### PR TITLE
Disable window view with analyzer properly

### PR DESCRIPTION
**Pull request by tavplubix**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

Fixes https://s3.amazonaws.com/clickhouse-test-reports/62266/578c4cfb9deeb023075695804f01ee100106115b/stress_test__msan_.html
```
+ echo -e 'Cannot start clickhouse-server\tFAIL\t\N\t [ 28209 ] {} <Error> Application: Caught exception while loading metadata: Code: 695. DB::Exception: Load job \'\''load table test_3.wv\'\'' failed: Code: 1. DB::Exception: Experimental WINDOW VIEW feature is not supported with new infrastructure for query analysis (the setting \'\''allow_experimental_analyzer\\n [ 28209 ] {} <Error> Application: Code: 695. DB::Exception: Load job \'\''load table test_3.wv\'\'' failed: Code: 1. DB::Exception: Experimental WINDOW VIEW feature is not supported with new infrastructure for query analysis (the setting \'\''allow_experimental_analyzer\'\''): Cannot attach table `test_3`.`wv` fro\\n [ 29235 ] {} <Error> Application: Caught exception while loading metadata: Code: 695. DB::Exception: Load job \'\''load table test_3.wv\'\'' failed: Code: 1. DB::Exception: Experimental WINDOW VIEW feature is not supported with new infrastructure for query analysis (the setting \'\''allow_experimental_analyzer\\n [ 29235 ] {} <Error> Application: Code: 695. DB::Exception: Load job \'\''load table test_3.wv\'\'' failed: Code: 1. DB::Exception: Experimental WINDOW VIEW feature is not supported with new infrastructure for query analysis (the setting \'\''allow_experimental_analyzer\'\''): Cannot attach table `test_3`.`wv` fro\\n [ 30111 ] {} <Error> Application: Caught exception while loading metadata: Code: 695. DB::Exception: Load job \'\''load table 01056_window_view_proc_hop_watch.wv\'\'' failed: Code: 1. DB::Exception: Experimental WINDOW VIEW feature is not supported with new infrastructure for query analysis (the setting \'\''a\\n [ 30111 ] {} <Error> Application: Code: 695. DB::Exception: Load job \'\''load table 01056_window_view_proc_hop_watch.wv\'\'' failed: Code: 1. DB::Exception: Experimental WINDOW VIEW feature is not supported with new infrastructure for query analysis (the setting \'\''allow_experimental_analyzer\'\''): Cannot atta\\n [ 31001 ] {} <Error> Application: Caught exception while loading metadata: Code: 695. DB::Exception: Load job \'\''load table 01056_window_view_proc_hop_watch.wv\'\'' failed: Code: 1. DB::Exception: Experimental WINDOW VIEW feature is not supported with new infrastructure for query analysis (the setting \'\''a\\n'
```

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

---
### Modify your CI run:
**NOTE:** If your merge the PR with modified CI you **MUST KNOW** what you are doing
**NOTE:** Checked options will be applied if set before CI RunConfig/PrepareRunConfig step

#### Include tests (required builds will be added automatically):
- [ ]  Fast test
- [ ]  Integration Tests
- [ ]  Stateless tests
- [ ]  Stateful tests
- [ ]  Unit tests
- [ ]  Performance tests
- [ ]  All with ASAN
- [ ]  All with TSAN
- [ ]  All with Analyzer
- [ ]  Add your option here

#### Exclude tests:
- [ ]  Fast test
- [ ]  Integration Tests
- [ ]  Stateless tests
- [ ]  Stateful tests
- [ ]  Performance tests
- [ ]  All with ASAN
- [ ]  All with TSAN
- [ ]  All with MSAN
- [ ]  All with UBSAN
- [ ]  All with Coverage
- [ ]  All with Aarch64
- [ ]  Add your option here

#### Extra options:
- [ ]  do not test (only style check)
- [ ]  disable merge-commit (no merge from master before tests)
- [ ]  disable CI cache (job reuse)

#### Only specified batches in multi-batch jobs:
- [ ]  1
- [ ]  2
- [ ]  3
- [ ]  4

### CODE REVIEW

**Comment by robot-clickhouse** [comment]
*This is an automated comment for commit 982d3db5b1fa220323c3f7d911b11b375945ca93 with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/62367/982d3db5b1fa220323c3f7d911b11b375945ca93/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>PR Check</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td>✅ success</td></tr>
<tbody>
</table>
</details>

**Comment by alexey-milovidov** [comment]
> Experimental WINDOW VIEW feature is not supported with new infrastructure for query analysis

Experimental WINDOW VIEW feature is not supported in the current infrastructure for query analysis.

File: src/Storages/WindowView/StorageWindowView.cpp [hunk 1]
```diff
@@ -1681,6 +1706,13 @@ StoragePtr StorageWindowView::getTargetTable() const
     return DatabaseCatalog::instance().getTable(target_table_id, getContext());
 }
 
+void StorageWindowView::throwIfWindowViewIsDisabled(ContextPtr local_context) const
+{
+    if (disabled_due_to_analyzer || (local_context && local_context->getSettingsRef().allow_experimental_analyzer))
+        throw Exception(ErrorCodes::UNSUPPORTED_METHOD, "Experimental WINDOW VIEW feature is not supported "
+                        "with new infrastructure for query analysis (the setting 'allow_experimental_analyzer')");
```
Comment (alexey-milovidov) [inline src/Storages/WindowView/StorageWindowView.cpp:1713]: .

**Comment by clickgapai** [comment]
Hi @tavplubix — while reviewing this PR I found the following:

- **Coverage gap:** `throwIfWindowViewIsDisabled` paths and `disabled_due_to_analyzer` ATTACH state are entirely untested — https://github.com/ClickHouse/ClickHouse/pull/104030

Could you review the test PR and add the `can be tested` label if it looks good?
Happy to discuss — close anything that's wrong or already addressed.

**Pull request merged**